src/3/world1.md

```diff
@@ -1,154 +1,155 @@
 
-"Don't worry, when it comes to Rinne, she can quickly..."
+"Don't worry. When it comes to Rinne, wherever I am, she can quickly..."
 
 _TL Note:_ TODO `リンネなら俺がどこにいてもすぐ──` I'm not sure how to re-order sentence to not miss the port about Kay's place
+_ED Note:_ How does the above wording sound?
 
 "Quickly?"
 
 "...She'll come back for sure.
-Rinne already gotten used exploring this forest."
+Rinne has already gotten used to exploring this forest."
 
-Kai wondered how Jeanne would react, if he'd just say that Rinne could pin down Kai's location with her nose.
+Kai wondered how Jeanne would react, if he'd just said that Rinne could pin down Kai's location with her nose.
 
-The girl, named Rinne, was in fact non-human.
-She had mix of blood from various human enemies.
-But among humans only Kai was aware of it.
+The girl named Rinne, was in fact non-human.
+She had a mix of blood from various human enemies.
+But among humans only Kai was aware of this fact.
 
 "By the way, Jeanne, weren't you supposed to attend a meeting in the village?"
 
 "That's right.
-Together with angels...
-Anyway, the elven elder got tired of waiting, just where is our Reiren?"
+Together with the angels...
+Anyway, the elven elder got tired of waiting... just where is our Reiren?"
 
-"Well then how about I'll go to bring her?
-I remember which direction, Rinne took."
+"Well then, how about I go fetch her?
+I remember which direction Rinne took."
 
-He pointed towards brushes deep in the forest.
+He pointed towards bushes deep in the forest.
 
 "Jeanne, are you going back?"
 
 "No, I'll tag along.
-I've been sitting the whole time while in discussion with foreign gods, my body feels stiff.
-So I feel like to take a short walk."
+I've been sitting the whole time while in discussion with the foreign gods, now my body feels stiff.
+So I feel like taking a short walk."
 
-They walked through animal trail.
-And while moving forward with withered grass under their foot.
+They walked through the animal trail,
+moving forward with withered grass under their feet.
 
-"Just now, I heard quite surprising story from elves."
+"Just now, I heard quite a surprising story from the elves."
 
-He wanted to ask what, but before that Jeanne made a sigh and continued.
+He wanted to ask what, but before that Jeanne heaved a sigh and continued.
 
-"So for foreign gods it might be normal to sit down for about a month without eating anything when they have serious discussion.
-Carry on, without sleeping or even a single drop of water."
+"So for the foreign gods it might be normal to sit down for about a month without eating anything when they have serious discussions.
+They tend to carry on, without sleeping or even a single drop of water."
 
 "I cannot imagine."
 
 "Yeah, hearing it made me shiver.
-And we were fighting against race that could calmly talk about something like that."
+And we were fighting against races that could calmly talk about something like that."
```

Comment:
`races` -> `the race`

I know it is confusing the way author made foreigns gods, but they are referred as singular race always

Reply:
Ah I see. My bad. I thought he was talking about all the other races as well!
"The race" is correct!